### PR TITLE
Make validTo field in consentRecord optional

### PR DESCRIPTION
**Pull request by stevenvegt**

### CODE REVIEW

**Comment by stevenvegt** [comment]
@woutslakhorst can you take a good look at my migrations? First time writing them and unsure if its correct. E.g. the `down` migration might not work when there are actual rows with `valid_to = null`.

File: migrations/4_alter_consent_record_make_valid_to_optional.down.sql [hunk 1]
```diff
@@ -0,0 +1,20 @@
+DROP INDEX uniq_record_version;
+
+ALTER TABLE consent_record RENAME TO consent_record_tmp;
+
+CREATE TABLE consent_record (
+    id INTEGER PRIMARY KEY AUTOINCREMENT,
+    patient_consent_id VARCHAR(255) REFERENCES patient_consent(id),
+    valid_from DATE NOT NULL,
+    valid_to DATE NOT NULL,
```
Comment (woutslakhorst) [inline migrations/4_alter_consent_record_make_valid_to_optional.down.sql]: Add a default value in 2100 or so.

Then the question is if you're allowed to do an INSERT with null value, does it error or does it use default?
If error, the insert needs to be separated into 2 queries: one with valid_to and one without.

**Review by woutslakhorst — CHANGES_REQUESTED** [review]
Still some red flags in the build. Coverage declined because of the extra if-statements. Should add some extra tests.

File: migrations/4_alter_consent_record_make_valid_to_optional.down.sql (hunk 1 above)
Comment (stevenvegt) [reply migrations/4_alter_consent_record_make_valid_to_optional.down.sql]: Done

**Comment by stevenvegt** [comment]
FYI: Codacy breaks because of generated bin-data file.

**Pull request merged**